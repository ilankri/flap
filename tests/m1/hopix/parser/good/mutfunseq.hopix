
fun f (x) = a; b

and g (x) = c; d

and h (x) = d; e

