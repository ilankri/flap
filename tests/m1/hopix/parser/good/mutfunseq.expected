
fun f (x) = val nothing = a ; b
and g (x) = val nothing = c ; d
and h (x) = val nothing = d ; e
